build.sbt

```diff
@@ -320,7 +320,9 @@ lazy val rosetta = (project in file("modules/rosetta"))
     libraryDependencies := Seq(
       CompilerPlugin.kindProjector,
       CompilerPlugin.betterMonadicFor,
-      CompilerPlugin.semanticDB
+      CompilerPlugin.semanticDB,
+        Libraries.doobieCore,
+    Libraries.doobieQuill,
```

Comment:
it's not required now, for auto generated classes

Reply:
Removed.